README.md

```diff
@@ -1,27 +1,57 @@
 # ![nf-core](nf-core-logos/nf-core-logo.png)
-# http://nf-co.re
 
-## [nf-core logos](nf-core-logos)
-- Countains original logo and icon in `ai` format, exported to `png` and also converted to `svg`
+## Repository contents
 
-## [make_logo](make_logo)
-- Countains a template logo with `GenericName` for a new pipeline, and a minimalist bash script to generate a new logo
+### [nf-core logos](nf-core-logos)
 
-### Requirements
-- Inkscape
-- [Maven Pro Bold](https://fonts.google.com/specimen/Maven+Pro) font.
+- Contains original logo and icon in `ai` format, exported to `png` and also converted to `svg`
+
+### [make_logo](make_logo)
+
+- Contains a template logo with `GenericName` for a new pipeline, and a minimalist bash script to generate a new logo
+
+## Create logo using Docker
+
+We provide a Docker Image on [DockerHub](https://cloud.docker.com/u/nfcore/repository/docker/nfcore/logos) that you may use to create logos for your pipeline easily.
+
+```bash
+docker run -v /path/on/host:/data -it nfcore/logos bash
+/make_logo.sh NewPipeline
+mv NewPipeline* /data
+```
+
+You will then find your logos in PNG and SVG format in `/path/on/host/`.
+
+## Create logo using Singularity
+
+You can also pull the Image from DockerHub using Singularity and do the same thing:
 
-Example:
+```bash
+singularity pull logo.sif docker://nfcore/logos
+singularity exec logo.sif /make_logo.sh NewPipeline
 ```
+
+## Create logo locally
+
+### Requirements
+
+The bash script needs to have two dependencies installed in order to be able to correctly generate the logo for you:
+
+- [Inkscape](https://inkscape.org/)
+- [Maven Pro Bold Fonts](https://fonts.google.com/specimen/Maven+Pro)
+
+#### Example
+
+```bash
 cd make_logo
 ./make_logo.sh NewPipeline
 ```
 
-Result:
-```
+#### Result:
+
+```bash
 NewPipeline_logo.png
 NewPipeline_logo.svg
-
 ```
 
 ![NewPipeline](make_logo/NewPipeline_logo.png)
```

Comment:
Was just thinking about this and it might be nice to put an incorrect image here too so people can see what the image is not supposed to look like. Maybe put the image above in another section called "## Correct logo", and add an incorrect version to "## Incorrect logo"?

Reply:
Could put a note in somewhere to say "Please double-check that the logo uses the same fonts as the image below. Its quite easy to miss!".

Reply:
Ah nice, I can do that